isb_lib/opencontext_adapter.py

```diff
@@ -76,6 +76,7 @@ def __init__(
             date_end=date_end,
             page_size=page_size,
         )
+        self._past_date_start = False
```

Comment:
New field to catch the situation we were in where we've reached the end of iteration due to being past the start date.